ios/Plugin/Plugin.swift

```diff
@@ -386,3 +395,32 @@ public class NativeBiometric: CAPPlugin {
         guard status == errSecSuccess || status == errSecItemNotFound else { throw KeychainError.unhandledError(status: status) }
     }
 }
+
+extension LAContext {
```

Comment:
This is to only add functionality to the existing class mas, since LA context doesn't include the method by default. Ref : https://docs.swift.org/swift-book/LanguageGuide/Extensions.html#:~:text=Extensions%20add%20new%20functionality%20to,to%20categories%20in%20Objective%2DC.

Reply:
I see, thank you for asnwering. Learning more about swift everyday :ok_hand: